### PR TITLE
LocalEvenniaTest - uses settings-defined typeclasses

### PR DESCRIPTION
**Pull request by trhr**
#### Brief overview of PR changes/additions
The EvenniaTest class hardcodes in DefaultObject, DefaultCharacter, etc as the spawned typeclasses. This inherited class uses the defaults as defined by settings.py

#### Motivation for adding to Evennia
Tests shouldn't break because my hooks don't run when objects are created. This could probably be extended further as a proper 'mygame tester'

#### Other
I know this is all perfectly doable without a core contrib, but anything that reminds people "hey, you can actually use unit tests by yourself even if you don't intend to contribute" is a good thing.

### CODE REVIEW

File: evennia/utils/test_resources.py [hunk 1]
```diff
@@ -157,3 +157,11 @@ def tearDown(self):
         self.account.delete()
         self.account2.delete()
         super().tearDown()
+
+class LocalEvenniaTest(EvenniaTest):
+    account_typeclass = settings.BASE_ACCOUNT_TYPECLASS
```
Comment (Griatch) [inline evennia/utils/test_resources.py:162]: This needs a documentation string to explain the purpose of this as a testing base.

**Comment by trhr** [comment]
Anything else?

**Comment by Griatch** [comment]
No this is probably fine, just haven't gotten around to PR-re-reviewing lately.

**Comment by trhr** [comment]
40% of my code coverage runs through this class and I haven't noticed anything weird.

**Comment by Griatch** [comment]
Looks straightforward enough. Merged.

**Pull request merged**